### PR TITLE
Add remaining Spanish translations

### PR DESCRIPTION
**Pull request by loft17**
Add remaining translations

Closes #

Changes proposed in this pull request:

-
-
-

How to test the feature manually:

1.
2.
3.

Pull request checklist:

- [ ] clear commit messages
- [ ] code manually tested
- [ ] unit tests written (optional if too hard)
- [ ] documentation updated

[Additional information can be found in the documentation](https://github.com/FreshRSS/FreshRSS/tree/master/docs/en/developers/04_Pull_requests.md).

### CODE REVIEW

**Review by Frenzie — APPROVED** [review]
lgtm

File: app/i18n/es/gen.php [hunk 1]
```diff
@@ -121,21 +121,21 @@
...
-		'pt-br' => 'Português (Brasil)',	// TODO - Translation
-		'ru' => 'Русский',	// TODO - Translation
-		'sk' => 'Slovenčina',	// TODO - Translation
-		'tr' => 'Türkçe',	// TODO - Translation
-		'zh-cn' => '简体中文',	// TODO - Translation
+		'pt-br' => 'Portugues (Brasil)',
+		'ru' => 'Ruso',
+		'sk' => 'Eslovaco',
+		'tr' => 'Turco',
+		'zh-cn' => 'Chino Simplificado',
```
Comment (aledeg) [inline app/i18n/es/gen.php:138]: should we translate those?

Comment (Frenzie) [reply app/i18n/es/gen.php:138]: I'd say probably not; in Dutch I and/or other translators purposefully didn't.

**Comment by aledeg** [comment]
@loft17 thank you for your contribution. Could you please not translate languages to keep it consistent with other languages. Those items are marked to be translated only because they are not listed in the `cli/i18n/ignore/es.php` file. You can check one of the existing files in the `cli/i18n/ignore` folder and duplicate it to match Spanish strings.

**Comment by Alkarex** [comment]
The original repo for the PR has apparently disappeared, so let's merge this one and fix in another PR

**Comment by Alkarex** [comment]
https://github.com/FreshRSS/FreshRSS/pull/2830

**Pull request merged**